src/java.desktop/share/classes/sun/swing/FilePane.java

```diff
@@ -528,6 +529,7 @@ protected void installDefaults() {
         viewTypeActionNames[VIEWTYPE_DETAILS] =
                         UIManager.getString("FileChooser.detailsViewActionLabelText", l);
 
+        byteString      = UIManager.getString("FileChooser.fileSizeBytes", l);
```

Comment:
is the '-' character misaligned or is this a github rendering artifact?

Reply:
I didn't get you. Can you please specify the line number ?